addons/crm/static/src/js/systray_activity_menu.js

```diff
@@ -0,0 +1,17 @@
+odoo.define('crm.systray.ActivityMenu', function (require) {
+"use strict";
+
+var ActivityMenu = require('mail.systray.ActivityMenu');
+
+ActivityMenu.include({
+    /**
+     * @override
+     */
+    _getViewsList(model) {
+        if (model === "crm.lead") {
+            return this._super.apply(this, arguments).concat([[false, 'calendar'], [false, 'pivot'], [false, 'graph'], [false, 'activity']]);
+        }
+        return this._super.apply(this, arguments);
```

Comment:
new short syntax (to change above too if we keep the `_super` depending on suggestion)
```suggestion
        return this._super(...arguments);
```